modules/zip/doc_classes/ZIPReader.xml

```diff
@@ -61,6 +61,16 @@
 				Must be called after [method open].
 			</description>
 		</method>
+		<method name="get_compression_level">
+			<return type="int" />
+			<param index="0" name="path" type="String" />
+			<param index="1" name="case_sensitive" type="bool" default="true" />
+			<description>
+				Returns the compression level of the file in the loaded zip archive.
+				Returns [code]-1[/code] if the file doesn't exist or any other error occurs.
+				Must be called after [method open].
```

Comment:
```suggestion
				Returns the compression level of the file in the loaded zip archive. Returns [code]-1[/code] if the file doesn't exist or any other error occurs. Must be called after [method open].
```
Each line is a new paragraph, think this is better this way

Reply:
Instead of 1 line I chose 2 to be consistent with the rest of the docs. I can update the other docs though if you prefer it all in one paragraph.

Reply:
updated to 1 line